app/controllers/stripe-setup/add-psp-account-details/get-controller.test.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+// NPM dependencies
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+const sinon = require('sinon')
+const proxyquire = require('proxyquire')
+const paths = require('../../../paths')
+
+// Global setup
+chai.use(chaiAsPromised)
+const { expect } = chai
+
+const getController = function getController (stripeAccountSetupResponse) {
+  return proxyquire('./get-controller', {
+    '../../../services/clients/connector_client': {
+      ConnectorClient: function () {
+        this.getStripeAccountSetup = (gatewayAccountId, correlationId) => {
+          return new Promise(resolve => {
+            resolve(stripeAccountSetupResponse)
+          })
+        }
```

Comment:
The powers that be in Javascript land anticipated the need to do this!

```suggestion
        this.getStripeAccountSetup = (gatewayAccountId, correlationId) => Promise.resolve(stripeAccountSetupResponse)
```